README.md

```diff
@@ -1,58 +1,173 @@
+| Multilingual site               | [![ehh]](https://www.getzola.org/documentation/content/multilingual/) | ![no]  | ![yes] | ![yes]  |
+| Image processing                | [![yes]](https://www.getzola.org/documentation/content/image-processing/) | ![no]  | ![yes] | ![yes]  |
+| Link checker                    | [![yes]](https://www.getzola.org/documentation/getting-started/cli-usage/#check) | ![no]  | ![no]  | ![yes]  |
+| Search                          | [![ehh]](https://www.getzola.org/documentation/content/search/) | ![no]  | ![no]  | ![yes]  |
+| Data files                      | [![yes]](https://www.getzola.org/documentation/templates/overview/#load-data) | ![yes] | ![yes] | ![no]   |
+| LiveReload                      | [![yes]](https://www.getzola.org/documentation/getting-started/cli-usage/#serve) | ![no]  | ![yes] | ![yes]  |
+
+**Notes:**
+
+- *Multilingual site*
+  - zola gets ![ehh] because despite having a basic translations system, it is being redesigned to evade its current limitations (see [discussion on the forum](https://zola.discourse.group/t/rfc-internationalization-system-rework/546))
+- *Search*
+  - zola gets ![ehh] because search engine is disabled by default in some languages, as explained [in the docs](https://www.getzola.org/documentation/content/multilingual/#configuration)
+
+### Content library
+
+|                                 | [Zola](https://getzola.org/)   | [Cobalt](https://cobalt-org.github.io/) | [Hugo](https://gohugo.io/)   | [Pelican](https://blog.getpelican.com/) |
+|:--------------------------------|:------:|:------:|:------:|:-------:|
 | Aliases                         | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Pagination                      | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Custom taxonomies               | ![yes] | ![no]  | ![yes] | ![no]   |
-| Search                          | ![yes] | ![no]  | ![no]  | ![yes]  |
-| Data files                      | ![yes] | ![yes] | ![yes] | ![no]   |
-| LiveReload                      | ![yes] | ![no]  | ![yes] | ![yes]  |
-| Netlify support                 | ![yes] | ![no]  | ![yes] | ![no]   |
-| Vercel support                  | ![yes] | ![no]  | ![yes] | ![yes]  |
+| Assets co-location              | [![ehh]](https://www.getzola.org/documentation/content/overview/#asset-colocation) | ![yes] | ![yes] | ![yes]  |
+| HTML shortcodes                 | [![yes]](https://www.getzola.org/documentation/content/shortcodes/) | ![no]  | ![yes] | ![yes]  |
+| Format-specific shortcodes      | [![yes]](https://www.getzola.org/documentation/content/shortcodes/) |   ?    |    ?   |    ?    |
+| Internal links                  | [![ehh]](https://www.getzola.org/documentation/content/linking/) | ![no]  | ![yes] | ![yes]  |
+| Warn or error for broken links  | [![no]](https://github.com/getzola/zola/issues/977#issuecomment-759725671)  | ![no]  | [![yes]](https://gohugo.io/content-management/cross-references/) | ? |
```

Comment:
Wrong link? zola definitely does link checking

Reply:
Maybe it's not explained well. I meant to make it configurable whether some situation (like a broken link) is an error or a warning, as explained in the corresponding hugo docs.

Could be rephrased. I added it after i saw issue #1367 so since i was mentioning this situation in #977 i thought to link there. Happy to rephrase or remove entirely before we have a more thoughtful conversation about errors/warnings in zola.